server.yml

```diff
@@ -217,8 +217,8 @@ web-root: "/"
 # - enable-login allows users to log in via the web app, or API
 # - enable-reservations allows users to reserve topics (if their tier allows it)
 #
-# enable-signup: false
-# enable-login: false
+enable-signup: true
+enable-login: true
```

Comment:
Why is it required?
Aren't we creating our logic for backend and frontend?

Reply:
Discussed In meet